odl/phantom/transmission.py

```diff
@@ -330,7 +349,28 @@ def transposeravel(arr):
 
         image[i] += f
 
-    return space.element(image)
+    if value_type == 'materials':
+        materials = np.zeros(space.size)
+        # csf
+        materials[(image > 1.043) & (image < 1.047)] = 1
+        # less_dense_sphere
+        materials[(image > 1.047) & (image < 1.048)] = 2
+        # brain
+        materials[(image > 1.048) & (image < 1.052)] = 3
+        # denser_sphere
+        materials[(image > 1.052) & (image < 1.053)] = 4
+        # blood
+        materials[(image > 1.053) & (image < 1.056)] = 5
+        # eye
+        materials[(image > 1.058) & (image < 1.062)] = 6
```

Comment:
gap between 1.056 and 1.058 - not that it makes a difference, but looks like a gotcha.